packages/cli/src/services/url.service.ts

```diff
@@ -0,0 +1,92 @@
+import { Service } from 'typedi';
+import config from '@/config';
+
+@Service()
+export class URLService {
+	private urls: {
+		base: string;
+		rest: string;
+		editor: string;
+		webhook: string;
+		oauth1Callback: string;
+		oauth2Callback: string;
+	};
+
+	// TODO: do this and webhookBaseUrl need to be separate
+	get baseUrl() {
+		return this.urls.base;
+	}
+
+	/** This is the base url for REST Api calls */
+	get restBaseUrl() {
+		return this.urls.rest;
+	}
+
+	/** This is the base url of the UI */
+	get editorBaseUrl() {
```

Comment:
perhaps we should rename this to `frontendUrl`, and `baseUrl` to `backendUrl`.

Reply:
Yes please!